lib/Table/Csv.ts

```diff
@@ -52,12 +53,21 @@ export default class Csv {
     url: string,
     columnMajor: boolean = false
   ): Promise<string[][]> {
-    return new Promise<string[][]>((resolve, reject) => {
-      papaparse.parse(url, {
-        ...getParseOptions(columnMajor, resolve, reject),
-        download: true
-      });
+    return loadWithXhr({ url }).then(csv => {
+      if (typeof csv === "string") {
+        return Csv.parseString(csv, columnMajor);
+      } else {
+        throw "Request failed";
+      }
     });
+    // There is currently a bug when using papaparse to fetch CSV URLs
+    // See: https://github.com/mholt/PapaParse/pull/832
+    // return new Promise<string[][]>((resolve, reject) => {
+    //   papaparse.parse(url, {
+    //     ...getParseOptions(columnMajor, resolve, reject),
+    //     download: true,
+    //   });
+    // });
```

Comment:
Perhaps a changelog entry to reflect this change

Reply:
Scratch this, I'm so sorry - I missed the last entry in the changelog that covered this!